packages/graphql-hooks-ssr/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.7",
   "description": "Server side rendering utils for graphql-hooks",
   "main": "index.js",
+  "types": "index.d.ts",
```

Comment:
you need to add this to the `files` prop

Reply:
There isn’t one in this package as we don’t bundle anything, everything is published